### ISSUE TITLE
Error migrating from Freeform

### ISSUE THREAD

**Issue by jmauzyk** [verbb/formie#717]
**Description**

Looking to migrate our forms from Freeform to Formie. Running into the following error when trying to migrate forms from Freeform:
`Exception: Call to undefined method Solspace\Freeform\Fields\Pro\ConfirmationField::isChecked()`

**Steps to reproduce**
1. Create a Freeform form with a confirmation field, the target of which is an email field.
2. Try to run the Freeform migration tool in Formie settings.

**Additional info**
- Plugin version:
  - Formie 1.5.6
  - Freeform Pro 3.12.11
- Craft version: Pro 3.7.28
- Multi-site: No

**Comment by engram-design** [comment]
Looks like the confirmation field wasn't ever setup to be migrated properly (instead was assumed it was an Agree field). Now Formie has support for matching fields, this can be migrated correctly and fixed at the same time. Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.6",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.5.7](https://github.com/verbb/formie/releases/tag/1.5.7)
